### ISSUE TITLE
Error: Could not open applet file

### ISSUE THREAD

**Issue by OatmealParmesan** [Massdrop/mdloader#46]
![image](https://user-images.githubusercontent.com/75916739/101999836-e04c3980-3cae-11eb-9c65-d604d6d174ef.png)

**Comment by jli97** [comment]
I'm wondering if renaming the aplet file to the "applet-mdflash.bin" would help but I dont wanna break anything

**Comment by okjustgo** [comment]
Hitting this error too.

```
C:\Users\mrkai\Downloads>mdloader_windows.exe --first --download massdrop_ctrl_md_configurator_1611350148_397892225.bin --restart
Massdrop Loader 1.05

Massdrop Loader  Copyright (C) 2018-2020 Massdrop Inc.
This program is Free Software and has ABSOLUTELY NO WARRANTY

Scanning for device for 60 seconds
.......
Device port: COM8 (SAMD51J18A)

Opening port 'COM8'... Success!
Found MCU: SAMD51J18A
Bootloader version: v2.20 Mar 27 2019 10:04:47 [ctrl]
Applet file: applet-mdflash.bin
Error: Could not open applet file: applet-mdflash.bin
Closing port... Success!
```

**Comment by okjustgo** [comment]
I tried renaming, but the darn tool keeps getting stuck at "Scanning for device for 60 seconds" and hangs, then I can't kill it through any means. Frustrating.

**Comment by staceb** [comment]
Changing into the directory that contains the mdloader executable and running the command from there worked for me. Might be worth a try!

**Comment by smeffs** [comment]
You will need also the applet-mdflash.bin file in the directory.

**Comment by OatmealParmesan** [comment]
> You will need also the applet-mdflash.bin file in the directory.

Of course. This issue occurs with the applet-mdflash.bin file in the current directory.

**Comment by vascobrown** [comment]
> Changing into the directory that contains the mdloader executable and running the command from there worked for me. Might be worth a try!

Thanks, @staceb! I was having the same issue this worked for me.

**Comment by OatmealParmesan** [comment]
So... when are we going to address this issue? It still persists.

**Comment by smeffs** [comment]
> > You will need also the applet-mdflash.bin file in the directory.
> 
> Of course. This issue occurs with the applet-mdflash.bin file in the current directory.

Just to rule out the obvious, have you tried it from the command prompt rather than from powershell?
